tests/coalaHTMLTest.py

```diff
@@ -27,22 +29,28 @@ def test_output_file(self):
         update_file = ""
         noupdate_file = ""
         with prepare_file(["#todo this is todo"], None) as (lines, filename):
-            execute_coala(coala_html.main,
-                          "coala-html",
-                          "-c", os.devnull,
-                          "-b", "LineCountBear",
-                          "-f", re.escape(filename),
-                          "--nolaunch")
+            p_update = Process(target=execute_coala, args=(coala_html.main,
```

Comment:
If we agree on not covering tests for spawned server then I will revert back to old approach (not using Process)